.eslintrc.js

```diff
@@ -53,24 +46,21 @@ module.exports = {
       },
 
       rules: {
-        // the TypeScript compiler already takes care of this and
-        // leaving it enabled results in false positives for interface imports
-        'no-unused-vars': 'off',
+        '@typescript-eslint/ban-ts-comment': 'warn',
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-ts-comment.md

Reply:
Let's do this in a follow up...